src/main/java/com/northconcepts/datapipeline/examples/reader/TeeReaderExample.java

```diff
@@ -0,0 +1,67 @@
+package com.northconcepts.datapipeline.examples.reader;
+
+import java.io.File;
+import java.time.LocalDateTime;
+import java.util.concurrent.TimeUnit;
+
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.core.Record;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.core.TeeReader;
+import com.northconcepts.datapipeline.job.Job;
+
+public class TeeReaderExample {
+
+	private static final int MAX_TRUCKS = 1;
+	private static final long MAX_PACKAGES = 20;
+	private static final long RECORD_DELAY_MILLISECONDS = TimeUnit.SECONDS.toMillis(0);
+
+	public static void main(String[] args) throws Throwable {
+		DataReader reader = new FakePackageReader(MAX_TRUCKS, MAX_PACKAGES, RECORD_DELAY_MILLISECONDS);
+
+		TeeReader teeReader = new TeeReader(reader, new FakePackageWriter(), true);
```

Comment:
No need to declare a `teeReader` since we're not doing anything special with it.  Let's just reassign the `reader` from above.

Reply:
sure, I got your point.